allensdk/brain_observatory/vbn_2022/metadata_writer/dataframe_manipulations.py

```diff
@@ -526,3 +529,38 @@ def _get_session_duration_from_behavior_session_ids(
         index=pd.Int64Index([x['behavior_session_id'] for x in durations],
                             name='behavior_session_id'))
     return durations
+
+
+def strip_substructure_acronym_df(
```

Comment:
I'm not sure why this function is needed. Can you not just do
```
df[col_name] = df[col_name].apply(strip_substructure_acronym, axis=1)
```

Reply:
Just running apply results in a series....
```
>>> import pandas as pd
>>> data = [{'a': 1, 'b': 2}, {'a': 2, 'b': 3}]
>>> df = pd.DataFrame(data=data)
>>> def fn(x):
...     return x**2
... 
>>> 
>>> new_df = df['a'].apply(fn)
>>> new_df
0    1
1    4
Name: a, dtype: int64
```

I'm just going to go back to my "implement a function to munge the structure acronym column." It was tested.

I really do not like pandas

Reply:
You need to assign the output to a column, not to the dataframe